src/pages/receiving/components/EditModalTable.ts

```diff
@@ -35,6 +35,24 @@ class Row extends BasePageModel {
     this.expiryDatePickerField = new DatePicker(page, 'Expiry', row);
     this.quantityShippedField = new TextField(page, 'Quantity shipped', row);
   }
+
+  get clearProductSelect() {
+    return this.row.locator(
+      '[class="css-16pqwjk-indicatorContainer react-select__indicator react-select__clear-indicator"]'
```

Comment:
Are you sure this css class is constant? Let's double check that this `16pqwjk` is not changing.